index.html

```diff
@@ -0,0 +1,137 @@
+          <input
+            class="herosection-input"
+            type="text"
+            name=""
+            id=""
+            placeholder="Enter your Email Address"
+          />
+          <button class="hero-section-button">Get Started</button>
+        </div>
+      </div>
+      <div class="services">
+        <div>
+          <div class="service-card">
+            <div class="service-image">
+              <img
+                src="https://th.bing.com/th/id/OIP.ofZc6YfimQ0mpHZIShtgsAHaF-?w=259&h=209&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+                alt=""
+              />
+            </div>
+            <div class="service-text">
+              <h2>Premium Materials</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+          <div class="service-card">
+            <div class="service-image">
+              <img
+                src="https://th.bing.com/th/id/OIP.ofZc6YfimQ0mpHZIShtgsAHaF-?w=259&h=209&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+                alt=""
+              />
+            </div>
+            <div class="service-text">
+              <h2>Fast Shipping</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+          <div class="service-card">
+            <div class="service-image">
+              <img
+                src="https://th.bing.com/th/id/OIP.ofZc6YfimQ0mpHZIShtgsAHaF-?w=259&h=209&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+                alt=""
+              />
+            </div>
+            <div class="service-text">
+              <h2>Quality Assurance</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="youtube-section">
+        <a href="https://youtu.be/y8Yv4pnO7qc">
+          <img
+            src="https://th.bing.com/th/id/OIP.bXrHc29g4Cy3E5QH5imMjQHaFf?w=223&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+            alt=""
+          />
+        </a>
+      </div>
+      <div class="pricing-options">
+        <div class="price-card">
+          <h2>Tenor Trombone</h2>
+          <h2 class="pricing-price">$600</h2>
+          <p class="pricing-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate,
+            dolor explicabo. Reiciendis impedit maiores voluptas nesciunt illum
+            officia doloribus aperiam.
+          </p>
+        </div>
+        <div class="price-card">
+          <h2>Tenor Trombone</h2>
+          <h2 class="pricing-price">$600</h2>
```

Comment:
I think it would be better to use an **h3** tag for the pricing since **h2** is already used

Reply:
Hi, @Samuelwanza!! 👋
- Here you have made a good suggestion politely with reasoning. 👍 

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.
- [ ] Kindly use reasoning `why` more precisely so that students can easily understand your suggestion.  Such as:
  - I think it would be better to use an h3 tag for the pricing since h2 is already used and is not better for the accessibility.